ledger/sandbox-classic/src/test/lib/scala/platform/sandbox/auth/UnsecuredServiceCallAuthTests.scala

```diff
@@ -0,0 +1,40 @@
+// Copyright (c) 2021 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package com.daml.platform.sandbox.auth
+
+import com.daml.ledger.api.auth.client.LedgerCallCredentials
+import com.daml.ledger.api.testing.utils.SuiteResourceManagementAroundAll
+import com.daml.platform.sandbox.SandboxRequiringAuthorization
+import com.daml.platform.sandbox.services.SandboxFixture
+import io.grpc.stub.AbstractStub
+import org.scalatest.flatspec.AsyncFlatSpec
+import org.scalatest.matchers.must.Matchers
+import org.scalatest.Assertion
+
+import scala.concurrent.Future
+
+trait UnsecuredServiceCallAuthTests
```

Comment:
Very nice. 👏🏻

Reply:
Thank you! I've just refactored those traits a bit to reduce code duplication. Please have a look if you're interested.